### ISSUE TITLE
Twisted tower, Slayers killin us in fly 

### ISSUE THREAD

**Issue by lemalestfait** [WarEmu/WarBugs#17142]
So Twisted Tower is a good sc, and now some peoples start to glitch it to kill us on fly for the top of the tower. Slayers not only usin aoe through the safe wall, now they glitch and are ON our top base. So with magnificent bugs, you take damage when you're flyin to the top, and you are dead when you pop on the top. [https://www.casimages.com/i/20061604313587896.jpg.html](url) [https://www.casimages.com/i/200616043136212364.jpg.html](url)

**Comment by lemalestfait** [comment]
The two first damage slayers used it to kill many,many players on the sc. That is a good exploit to win.

**Comment by IceandFire1985** [comment]
Yep... its glitch/hax/cheat when order use it. But when choppa/mara/sorc/msquig do the same AND choppa GTTC u through the barrier its work as intended

**Comment by lemalestfait** [comment]
So you use this cheat cause you see others usin it too ? You're helpin a lot the RoR community. Thanks for feedback.

**Comment by Kaelang** [comment]
Thank you for reporting this. We'll take a look.
